packages/flutter_tools/test/commands.shard/permeable/analyze_once_test.dart

```diff
@@ -176,7 +196,7 @@ void bar() {
         tryToDelete(tempDir);
```

Comment:
Maybe use a different variable name to distinguish from the `tempDir` in the outer scope.

Reply:
I re-grouped things so there's no more shadowing/access to unused variables.

Reply:
I re-shuffled things so that there is no more shadowing of this variable (but didn't rename it).

I've also made it so we don't need to call pub anymore in this test.